plugins/woocommerce/src/Admin/Settings/SettingsSection.php

```
@@ -0,0 +1,38 @@
+	 */
+	public function get_script_handles( \WC_Settings_Page $parent_page ): array {
+		return array();
+	}
+
+	/**
+	 * Get the default save adapter for fields in this section.
+	 *
+	 * @param \WC_Settings_Page $parent_page Parent settings page.
+	 * @return string
+	 */
+	public function get_save_adapter( \WC_Settings_Page $parent_page ): string {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$parent_page'. (undefined)

(UnusedFormalParameter)